src/components/MultiHopTrade/hooks/useGetTradeQuotes/useGetTradeQuotes.tsx

```diff
@@ -64,14 +65,24 @@ const getMixPanelDataFromApiQuotes = (quotes: ApiQuote[]): GetMixPanelDataFromAp
   return { quoteMeta, sellAssetId, buyAssetId, sellAmountUsd }
 }
 
-const isEqualExceptAffiliateBps = (
+const isEqualExceptAffiliateBpsAndSlippage = (
```

Comment:
`slippageTolerancePercentage` is now an additional that may change, but shouldn't reset swappers.